tools/lsp/language/completion.rs

```diff
@@ -1204,7 +1206,6 @@ mod tests {
             "component Bar in🔺 Window {}",
         ];
         for source in sources {
-            eprintln!("Test for inherits in {source:?}");
```

Comment:
why remove this?
If the test fails, this tells what iteration of the loop caused the failure.
If the test doesn't fail, the output is being captured and not shown.